practice/00-HTML-CSS-basics/First exercise.html

```diff
@@ -0,0 +1,7 @@
+<!DOCTYPE html>
+<html>
+<body>
+<head>Hello World
```

Comment:
<head></head> is not a header. You can read about headers and the difference with <head> here: http://www.w3schools.com/html/html_headings.asp